src/renderer/template.ts

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { render } from "./renderer";
+import Path from 'path';
+import Fs from 'fs';
+import {transform} from '@babel/core';
+const ROOT_DIR = Path.resolve(__dirname, '../..');
+import {rollup} from 'rollup';
+import typescript from '@rollup/plugin-typescript';
+import commonjs from '@rollup/plugin-commonjs';
+import babel from '@rollup/plugin-babel';
+/**
+ * Imports a given file and return the imported component
+ * 
+ * @param filepath to import
+ */
+function importComponent(filepath:string): Promise<React.ReactElement> {
+  return new Promise((resolve) => {
+    import(filepath).then(component => { resolve(component.default()) })
+  })
+}
+
+async function transpileFilesRollup(filepath: string, outputDir: string){
+  const parsedFile = Path.parse(filepath)
+  const dir = parsedFile.dir
+  const files = Fs.readdirSync(dir);
+  if (!Fs.existsSync(outputDir)){
+    Fs.mkdirSync(outputDir);
+  }
```

Comment:
Let's not forget to change these functions to their asynchronous alternative.

Reply:
Will do :+1: